pkg/tsdb/influxdb/influxdb.go

```diff
@@ -158,14 +212,24 @@ func (e *Executor) createRequest(ctx context.Context, dsInfo *models.DataSource,
 
 	req.URL.RawQuery = params.Encode()
 
-	if dsInfo.BasicAuth {
-		req.SetBasicAuth(dsInfo.BasicAuthUser, dsInfo.DecryptedBasicAuthPassword())
-	}
-
 	if !dsInfo.BasicAuth && dsInfo.User != "" {
-		req.SetBasicAuth(dsInfo.User, dsInfo.DecryptedPassword())
+		req.SetBasicAuth(dsInfo.User, dsInfo.Password)
```

Comment:
so here, we don't need to set basic authentication in the request anymore, see pr: https://github.com/grafana/grafana-plugin-sdk-go/pull/379

the basic auth is covered eithter by sdk with user/password for legacy, and basicauthuser/basicauthpassword. you could remove this.

Reply:
didn't know that, will remove it